### ISSUE TITLE
GEO::mesh_make_atlas() very slow

### ISSUE THREAD

**Issue by cdcseacave** [BrunoLevy/geogram#38]
I finally was able to call and extract all data I need from `GEO::mesh_make_atlas()` however when trying it on any real mesh (number of faces > 500k) the method becomes very slow, basically unusable in any real wold application. Is there something I do wrong in compiling the library or the function in known to be slow?

Update: I tested the functionality in [GraphiteThree](https://github.com/BrunoLevy/GraphiteThree) and same problem.

**Comment by cdcseacave** [comment]
sounds great, it would be of great help if you can add that missing part too; thank you!

**Comment by BrunoLevy** [comment]
Just for my own curiosity, what is your usage scenario / application ?

**Comment by cdcseacave** [comment]
mesh texturing in a photogrammetry pipeline like OpenMVS

**Comment by cdcseacave** [comment]
I also get a lot of messages like this, is it normal?
```
| o-[ABF++       ] Warning: Could not initialize SuperLU extension            |
OpenNL error in nlOpenDLL(): Not implemented
```

**Comment by BrunoLevy** [comment]
Some news:
- New mesh segmentation is scheduled for v1.8.3 (will be there before the end of this year if everything goes well)
- About `nlOpenDLL()` error, on which OS are you running ?

**Comment by cdcseacave** [comment]
Thank you for the update!
I implemented my own mesh segmentation and parametrization, so no hurry on this.
I'm testing on Windows 10 x64 with vcpkg port (I was not able to build from the git sources). So it would be great if you have a look at the vcpkg port and update it: https://github.com/microsoft/vcpkg/tree/master/ports/geogram

**Comment by BrunoLevy** [comment]
I have no experience with vcpkg, but I'll take a look ...
What happened when you tried to buid from git sources ?
Did you follow the protocol explained here: https://github.com/BrunoLevy/geogram/wiki/compiling_Windows ?

**Comment by cdcseacave** [comment]
I do not remember what went wrong, but I tried now again the instructions in your link and it worked
though it would be even better if you can integrated it in vcpgk

**Comment by BrunoLevy** [comment]
Fixed in v1.8.3rc that has new mesh_make_atlas() function 
(github repo up to date if you want to test)

**Comment by cdcseacave** [comment]
thank you!
